src/pykeen/training/lcwa.py

```diff
@@ -23,8 +23,6 @@
 
 logger = logging.getLogger(__name__)
 
-name_to_index = {name: index for index, name in enumerate("hrt")}
```

Comment:
This was quite strange - everywhere else we use "head" / "relation" / "tail" (via the corresponding constants) instead of the single-letter variant.

Reply:
it would be good to have a single `Literal["h", "r", "t"]` in the pykeen.typing module to reuse. Might already exist 😅